velox/type/UnscaledLongDecimal.h

```diff
@@ -54,11 +54,19 @@ add(int128_t x, const int128_t y) {
 
 struct UnscaledLongDecimal {
  public:
+  inline static bool valueInRange(int128_t value) {
+    return (value >= kMin) && (value <= kMax);
+  }
+
   // Default required for creating vector with NULL values.
   UnscaledLongDecimal() = default;
 
-  constexpr explicit UnscaledLongDecimal(int128_t value)
-      : unscaledValue_(value) {}
+  explicit UnscaledLongDecimal(int128_t value) : unscaledValue_(value) {
+    VELOX_CHECK(
```

Comment:
This code may run in hot loops, no? Should this be a DCHECK?

Reply:
Done